app/main.py

```diff
@@ -14,4 +14,11 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    phrase = phrase.lower()
+    letter = letter.lower()
+    for i in range(len(phrase)):
+        if phrase[i] == letter:
+            count += 1
```

Comment:
I can suggest a simple way.
You can create a list from the string:
list(string)
And in the list we have method .count